docs/yaml/elements/feature_identifier.yaml

```diff
@@ -0,0 +1,10 @@
+_name: FeatureIdentifier
+csv-header-name: feature_identifier
+tags:
+- _name: Index
+  csv-header-name: index
+  csv-type: xs:string
```

Comment:
WDYT about making the index field an integer? When this used to be called "ShapeIdentifier", I originally chose a string format to offer greater flexibility for other shapefile formats. But since we plan to extend the FeatureIdentifier approach using other additional fields within the FeatureIdentifier element in the future (e.g. add an attribute field), we'll get flexibility that way instead. So this seems reasonable to me to be an integer field.

Reply:
SGTM! Thanks. Update forthcoming.